numpy/core/include/numpy/npy_3kcompat.h

```diff
@@ -546,4 +546,10 @@ NpyCapsule_Check(PyObject *ptr)
 }
 #endif
 
+/* Introduced in https://github.com/python/cpython/commit/d2ec81a8c99796b51fb8c49b77a7fe369863226f */
```

Comment:
This might look better up top with the similar compatibility hack.

Reply:
I should also add a reference for the other one.